bundles/org.openhab.ui.dashboard/src/main/java/org/openhab/ui/dashboard/internal/DashboardServlet.java

```diff
@@ -180,4 +202,24 @@ private String getPackage() {
         }
         return null;
     }
+
+    private String replaceKeysWithLocaleFunction(String template, Locale locale) {
+        return replaceKeysWithFunction(template, (key) -> localizeFunction.apply(key, locale));
+    }
+
+    private String replaceKeysFromMap(String template, Map<String, String> map) {
+        return replaceKeysWithFunction(template, (key) -> map.getOrDefault(key, "\\${" + key + '}'));
```

Comment:
I found the issue. This line should be replace with:
```
return replaceKeysWithFunction(template, (key) -> Matcher.quoteReplacement(map.getOrDefault(key, "\\${" + key + '}')));
```

Reply:
It solves the previous problem but unfortunately it adds a \ in front of any replaced text.

Reply:
Doh! I must have been sleeping already.. I was looking at the big red box and totally missed the slashes....